spec/acceptance/suites/default/11_yum_internet_spec.rb

```diff
@@ -87,7 +42,7 @@
         'pupmod-simp-simp',
         'pupmod-simp-simplib',
       ]
-      hosts.each do |host|
+      block_on(hosts, :run_in_parallel => true) do |host|
```

Comment:
Please make these a variable that we can toggle via an environment variable. I've had cases that I couldn't troubleshoot without disabling parallelization by hand everywhere and that wasn't a huge deal of fun.